drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -622,8 +623,8 @@ static int stop_gps(struct device *dev, bool is_timeout)
 	retval = nrf_setsockopt(drv_data->socket,
 				NRF_SOL_GNSS,
 				NRF_SO_GNSS_STOP,
-				NULL,
-				0);
+				&delete_mask,
+				sizeof(uint32_t));
```

Comment:
`sizeof(nrf_gnss_delete_mask_t)` or `sizeof(delete_mask)` would be a better fit here - no silent assumption that `nrf_gnss_delete_mask_t` is `uint32_t`.

Reply:
true!